adafruit_bluefruitspi.py

```diff
@@ -166,12 +178,19 @@ def send_keyboard_code(self, evt):
             self._fifo_buffer.enqueue(k)
 
     def pop_keyboard_code_queue(self):
+        """Send an SDEP packet from the FIFO buffer to the Bluefruit"""
         data = self._fifo_buffer.dequeue()
         if data is not None:
```

Comment:
```
************* Module adafruit_bluefruitspi
adafruit_bluefruitspi.py:183:4: R0201: Method could be a function (no-self-use)
```
I'm not sure what the issue is here, this line isn't even pointing to a method definition, and the method it's within definitely refers to `self`

Reply:
hmm not sure either.. @dhalbert any idea

Reply:
`black` also ran and shifted the line numbering a bit. I think its referring to `_create_sdep_raw` which doesn't use self. You can remove self and then mark the function with `@staticmethod`.

Reply:
Thanks! That was the problem